openslides/agenda/static/js/agenda/base.js

```diff
@@ -112,61 +112,89 @@ angular.module('OpenSlidesApp.agenda', ['OpenSlidesApp.users'])
                     }
                 },
                 // override project function of jsDataModel factory
-                project: function() {
-                    return $http.post(
-                        '/rest/core/projector/1/prune_elements/',
-                        [{name: this.content_object.collection, id: this.content_object.id}]
-                    );
+                project: function (projectorId, tree) {
+                    var isProjectedId = this.isProjected(tree);
+                    if (isProjectedId > 0) {
+                        // Deactivate
+                        $http.post(
+                            '/rest/core/projector/' + isProjectedId + '/prune_elements/', []
+                        );
+                    }
+                    // Activate, if the projector_id is a new projector.
+                    if (isProjectedId != projectorId) {
+                        var name = tree ? 'agenda/item-list' : this.content_object.collection;
+                        var id = tree ? this.id : this.content_object.id;
+                        return $http.post(
+                            '/rest/core/projector/' + projectorId + '/prune_elements/',
+                            [{name: name, tree: tree, id: id}]
+                        );
+                    }
                 },
                 // override isProjected function of jsDataModel factory
-                isProjected: function (list) {
-                    // Returns true if there is a projector element with the same
-                    // name and the same id.
-                    var projector = Projector.get(1);
-                    var isProjected;
-                    if (typeof projector !== 'undefined') {
-                        var self = this;
-                        var predicate = function (element) {
-                            var value;
-                            if (typeof list === 'undefined') {
-                                // Releated item detail slide
-                                value = element.name == self.content_object.collection &&
-                                    typeof element.id !== 'undefined' &&
-                                    element.id == self.content_object.id;
-                            } else {
-                                // Item list slide for sub tree
-                                value = element.name == 'agenda/item-list' &&
-                                    typeof element.id !== 'undefined' &&
-                                    element.id == self.id;
-                            }
-                            return value;
-                        };
-                        isProjected = typeof _.findKey(projector.elements, predicate) === 'string';
-                    } else {
-                        isProjected = false;
+                isProjected: function (tree) {
+                    // Returns the projector id if there is a projector element with the same
```

Comment:
Can I create an Issue for this? I think this is some kind of feature for new plugins, and the possibility to project things multiple times should be implemented in the master. But not with this PR, because this was not specified at the beginning and not requested by the customer.